packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -96,35 +112,48 @@ popd
 
 %install
 rm -rf %{buildroot}
-mkdir -p %{buildroot}/%{_sysconfdir}/gofer/plugins
-mkdir -p %{buildroot}/%{_prefix}/lib/gofer/plugins
+mkdir -p %{buildroot}%{_sysconfdir}/gofer/plugins
+mkdir -p %{buildroot}%{_prefix}/lib/gofer/plugins
+
+cp etc/gofer/plugins/katelloplugin.conf %{buildroot}%{_sysconfdir}/gofer/plugins
+cp src/katello/agent/katelloplugin.py %{buildroot}%{_prefix}/lib/gofer/plugins
 
-cp etc/gofer/plugins/katelloplugin.conf %{buildroot}/%{_sysconfdir}/gofer/plugins
-cp src/katello/agent/katelloplugin.py %{buildroot}/%{_prefix}/lib/gofer/plugins
+%if %{dnf_install}
+%define katello_libdir %{python3_sitelib}/katello
+%define plugins_dir %{python3_sitelib}/dnf-plugins
+%define plugins_confdir %{_sysconfdir}/dnf/plugins
+%else
+%define katello_libdir %{python_sitelib}/katello
+%define plugins_dir %{_prefix}/lib/yum-plugins
+%define plugins_confdir %{_sysconfdir}/yum/pluginconf.d
+%endif
+
+mkdir -p %{buildroot}%{katello_libdir}
+mkdir -p %{buildroot}%{plugins_dir}
+mkdir -p %{buildroot}%{plugins_confdir}
 
-mkdir -p %{buildroot}/%{_prefix}/lib/yum-plugins
-cp src/yum-plugins/package_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
-cp src/yum-plugins/enabled_repos_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
+cp src/katello/*.py %{buildroot}%{katello_libdir}/
+cp etc/yum/pluginconf.d/*.conf %{buildroot}%{plugins_confdir}/
 
-mkdir -p %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/
-cp etc/yum/pluginconf.d/package_upload.conf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/package_upload.conf
-cp etc/yum/pluginconf.d/enabled_repos_upload.conf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/enabled_repos_upload.conf
+%if %{dnf_install}
+cp src/dnf_plugins/*.py %{buildroot}%{plugins_dir}/
```

Comment:
Technically this line could be before the `%if` since it's in both branches but it's not that important.

Reply:
I'll leave it :)